objects/obj_controller/Step_0.gml

```diff
@@ -47,43 +47,33 @@ if (sound_in=0) and (sound_to!=""){audio_stop_all();
 
 
 
-/*if (x<800) then x=800;
-if (y<450) then y=450;
-if (x>(room_width-800)) then x=room_width-800;
-if (y>(room_height-450)) then y=room_height-450;*/
-if (cheatcode != "")
-{
-}
-if (cheatcode == "req" && global.cheat_req == 0)
-{
+if (cheatcode != "") { 
+cheatyface = true }
```

Comment:
please put the closing bracket on a separate line, we seem to have decided on [K&R style](https://en.wikipedia.org/wiki/Indentation_style#K&R_style) for indentation